packages/serverless/src/apollo/app/collection.ts

```diff
@@ -108,9 +108,9 @@ const collectionCreate = authorizeWrapper(
         args.databaseName,
         args.collectionName,
         ctx.userName,
-        args.groupName,
         args.schema,
-        args.permissions,
+        args.groupName,
```

Comment:
`groupName` and `username` should change to: `owner` and `group`

Reply:
we need to wait for the common package to consist all these thing, the groupName now is messed up